### DISCUSSION TITLE
How to align individual table columns

### DISCUSSION THREAD

**Discussion by songololo** [typst/typst#1085] [Q&A] [answered]
Is there a way to set different forms of alignment for individual table columns?

I've tried a tuple of values but this doesn't seem to work.

Thanks.

```
#table(
    columns: (auto, auto, auto),
    align: (center),
   ...
)
```

**Comment by jassler** [comment, marked as answer]
Would [this](https://typst.app/docs/reference/layout/table#parameters--align) work for you?

```
#table(
    columns: (auto, auto, auto),
    align: (x, y) => (left, center, right).at(x),
    ...
)
```

**Reply by songololo** [reply]
Thanks, this works.
